### PR TITLE
fix_bug_lwm2m tests dif Port

### PR DESCRIPTION
**Pull request by nickAS21**
## Pull Request description

```
2024-04-10 16:44:48,708 [CoapServer(main)#7] ERROR o.t.s.t.l.s.d.AbstractLwM2MIntegrationDiffPortTest - RegistrationUpdate arguments [RegistrationUpdate [registrationId=3OhZexS2EL, clientTransportData=IpPeer [peerAddress=/127.0.0.1:58668, identity=Identity [psk=SOME_PSK_ID_5]], lifeTimeInSec=null, smsNumber=null, bindingMode=null, objectLinks=null, alternatePath=null, supportedContentFormats=null, supportedObjects=null, availableInstances=null, additionalAttributes={}, applicationData={}]]
```

## General checklist

- [x] You have reviewed the guidelines [document](https://docs.google.com/document/d/1wqcOafLx5hth8SAg4dqV_LV3un3m5WYR8RdTJ4MbbUM/edit?usp=sharing).
- [x] [Labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#about-labels) that classify your pull request have been added.
- [x] The [milestone](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones) is specified and corresponds to fix version.  
- [x] Description references specific [issue](https://github.com/thingsboard/thingsboard/issues).
- [x] Description contains human-readable scope of changes.
- [x] Description contains brief notes about what needs to be added to the documentation.
- [x] No merge conflicts, commented blocks of code, code formatting issues.
- [x] Changes are backward compatible or upgrade script is provided.
- [x] Similar PR is opened for PE version to simplify merge. Crosslinks between PRs added. Required for internal contributors only.

## Back-End feature checklist

- [x] Added corresponding unit and/or integration test(s). Provide written explanation in the PR description if you have failed to add tests.
- [x] If new dependency was added: the dependency tree is checked for conflicts.
- [x] If new service was added: the service is marked with corresponding @TbCoreComponent, @TbRuleEngineComponent, @TbTransportComponent, etc.
- [x] If new REST API was added: the RestClient.java was updated, issue for [Python REST client](https://github.com/thingsboard/thingsboard-python-rest-client) is created.
- [x] If new yml property was added: make sure a description is added (above or near the property).

### CODE REVIEW

**Review by smatvienko-tb — COMMENTED** [review]
Please move the mock to the beginning of the method. before you start working with the lwm2m client

see the reference: https://github.com/thingsboard/thingsboard-pe/pull/2501

File: common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/store/TbInMemoryRegistrationStore.java [hunk 1]
```diff
@@ -147,6 +147,8 @@ public Deregistration addRegistration(Registration registration) {
 
     @Override
     public UpdatedRegistration updateRegistration(RegistrationUpdate update) {
+        // test fix bug Diff port
+        log.warn("updateRegistration inMemory {}", update);
```
Comment (smatvienko-tb) [inline common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/store/TbInMemoryRegistrationStore.java:151]: log.trace

File: common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/store/TbLwM2mRedisRegistrationStore.java [hunk 2]
```diff
@@ -222,6 +222,8 @@ public Deregistration addRegistration(Registration registration) {
 
     @Override
     public UpdatedRegistration updateRegistration(RegistrationUpdate update) {
+        // test fix bug Diff port
+        log.warn("updateRegistration Redis {}", update);
```
Comment (smatvienko-tb) [inline common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/store/TbLwM2mRedisRegistrationStore.java:226]: log. trace

**Review by smatvienko-tb — APPROVED** [review]
good progress!

**Review by smatvienko-tb — APPROVED** [review]
Good job!

File: application/src/test/resources/logback-test.xml [hunk 3]
```diff
@@ -37,6 +37,9 @@
 
     <!-- Device actor message processor debug for the test scope -->
     <!--    <logger name="org.thingsboard.server.actors.device.DeviceActorMessageProcessor" level="DEBUG" />-->
+    <logger name="org.thingsboard.server.transport.lwm2m.server.store.TbInMemoryRegistrationStore" level="TRACE"/>
```
Comment (ashvayka) [inline application/src/test/resources/logback-test.xml:40]: Please revert unnecessary log changes.

Comment (nickAS21) [reply application/src/test/resources/logback-test.xml:40]: Ok

**Pull request merged**